the_destination_of_shadow/Character.h

```diff
@@ -19,27 +20,20 @@ enum Direction
 /**
 * @brief PlayerとEnemyの親class
 */
-class CharacterBase
+class CharacterBase : public Object
 {
 protected:
+
 	INT m_hp;
 	FLOAT m_move_speed;
 	Direction m_save_direction;
 
-	TEX m_tex;
-
-	FLOAT m_x;
-	FLOAT m_y;
-	FLOAT m_x_width;
-	FLOAT m_y_height;
-
-	FLOAT m_tu;
-	FLOAT m_tv;
-	FLOAT m_tu_width;
-	FLOAT m_tv_height;
-
 public:
-	virtual VOID Move()=0;
+
+	/**
+	* @brief キャラクターの動作、エネミーならAI、プレイヤーなら動作に関する処理を書く
```

Comment:
書くとするなら
```suggestion
	* @brief キャラクターを動かす
	* @note 各派生クラスにて、動作に関する処理をimplementすること
```
とかかなぁ。
briefに書く内容では無いですね。

Reply:
noteの存在を忘れてました。
修正します